src/index.d.ts

```diff
@@ -0,0 +1,177 @@
+export interface IEnumerable<T> extends Iterable<T> {
+  aggregate(initialValue: T, aggregateFunction: (accumulator: T, element: T, index: number) => T): T;
+  aggregate<TResult>(
+    initialValue: TResult,
+    aggregateFunction: (accumulator: TResult, element: T, index: number) => T
+  ): TResult;
+
+  all(predicate: (element: T) => boolean): boolean;
+
+  any(predicate: (element: T) => boolean): boolean;
```

Comment:
It is necessary to add declaration for overload without parameters.
```suggestion
  any(): boolean;
  any(predicate: (element: T) => boolean): boolean;
```